resources/fetching/fetchers/azure/assets_fetcher.go

```diff
@@ -42,6 +42,7 @@ type AzureResource struct {
 var AzureResourceTypes = map[string]string{
 	inventory.VirtualMachineAssetType: fetching.AzureVMType,
 	inventory.StorageAccountAssetType: fetching.AzureStorageAccountType,
+	inventory.WebSitesAssetType:       fetching.AzureWebSiteType,
```

Comment:
IMO we should inline all the asset type strings and generate the sub type (or type) from the string itself. same as we did in GCP